pom.xml

```diff
@@ -120,6 +122,26 @@
                 <artifactId>farao-glsk-document-api</artifactId>
                 <version>${farao.core.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.farao-community.farao</groupId>
+                <artifactId>farao-crac-creator-api</artifactId>
+                <version>${farao.core.version}</version>
+            </dependency>
```

Comment:
This should not be necessary as it is already included in farao-parent-bom

Reply:
yes